frontend/src/store/questions/index.js

```diff
@@ -43,9 +43,6 @@ const mutations = {
   API_ERROR (state, {code, message}) {
     state.error.code = code
     state.error.message = message
-  },
```

Comment:
I suppose this makes sense to always rely on server as source of truth, seems kinda odd to be rerendering the whole list on a single question change tho.